Bundle/WidgetMapBundle/Resources/config/services.yml

```diff
@@ -5,9 +5,17 @@ services:
     victoire_widget_map.widget_data_warmer:
         class: Victoire\Bundle\WidgetMapBundle\Warmer\WidgetDataWarmer
         arguments:
-            - "@annotation_reader"
-            - "@victoire_view_reference.repository"
-            - ["\Victoire\Bundle\MediaBundle\Entity\Media"]
+            - @annotation_reader
+            - @victoire_view_reference.repository
+            - @victoire_widget.widget_helper
+            - WidgetSliderItem: WidgetSlider
```

Comment:
I disagree with this point of view.
Even if:
- it "don't care" about a Missing class name (eg. the WidgetSlider is not installed in this project)
- it's not a big deal (indeed) if a new widget appears and we didn't change these lines

the right place to add WidgetSlider's extra configuration is in the widget's configuration, not outside and this is the kind of things we need to avoid to stop code pollution. I insist to:

- start a new markdown file like doc/contributing/performance.md and explane what could think a new widget's creator
- move the logic as I said in the concerned Bundle's configuration.

@paulandrieux @alexislefebvre @BFoucher @MadeWilson ?